doc/users/next_whats_new/asinh_scale.rst

```diff
@@ -0,0 +1,32 @@
+New axis scale ``asinh``
+------------------------
+
+The new ``asinh`` axis scale offers an alternative to ``symlog`` that
+smoothly transitions between the quasi-linear and asymptotically logarithmic
+regions of the scale. This is based on an arcsinh transformation that
+allows plotting both positive and negative values that span many orders
+of magnitude.
```

Comment:
I guess the way to get this in sooner rather than later is to mark it all experimental so that we have more freedom to change the locator and formatter after it has gone in.  Can you do that here, the docstring, and the example?  Then feel free to ping for a review on a weekly basis - we should be able to get this in.

Reply:
Thanks, Jody.
I'm happy to add "experimental" markings. Is there a standard format for doing this? Is this just a matter of adding clauses like `Warnings - this method is experimental as of 3.x, and the API may change` to the docstrings?

Reply:
I think that's all there is to it.  You could see what subplot_mosaic says for a template.